### PR TITLE
add decision_letter_sent field to claims and appeals overview

### PR DESCRIPTION
**Pull request by aherzberg**
## Summary
FE is adding a tag to index page of claims and appeals, this adds a field that will allow them to check if a claim or appeal has a decision letter. 

## Related issue(s)
https://github.com/department-of-veterans-affairs/va-mobile-app/issues/5339

## Testing done

- *Describe what the old behavior was prior to the change*
- *Describe the steps required to verify your changes are working as expected*
- *Describe the tests completed and the results*

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: modules/mobile/app/models/mobile/v0/adapters/claims_overview.rb [hunk 1]
```diff
@@ -65,7 +66,8 @@ def parse_appeal(entry)
               completed: !entry['attributes']['active'],
               date_filed: entry['attributes']['events'][filed_index]['date'],
               updated_at: entry['attributes']['events'].last['date'],
-              display_title: get_appeals_display_title(subtype, entry['attributes']['programArea'])
+              display_title: get_appeals_display_title(subtype, entry['attributes']['programArea']),
+              decision_letter_sent: false
```
Comment (aherzberg) [inline modules/mobile/app/models/mobile/v0/adapters/claims_overview.rb:70]: appeal decision letters aren't included in decision letter listing right now. Will be added later.

**Review by kpethtel — DISMISSED** [review]
👍

File: modules/mobile/app/models/mobile/v0/contracts/claims_and_appeals.rb [hunk 2]
```diff
@@ -7,6 +7,7 @@ class ClaimsAndAppeals < PaginationBase
         params(Schemas::DateRangeSchema) do
           optional(:use_cache).maybe(:bool, :filled?)
           optional(:show_completed).maybe(:bool, :filled?)
+          optional(:decision_letter_sent).maybe(:bool, :filled?)
```
Comment (kpethtel) [inline modules/mobile/app/models/mobile/v0/contracts/claims_and_appeals.rb:10]: Is this necessary? These contracts are for validating incoming params from the mobile client? Unless we're planning to filter based on this value, I doubt this is a controller param.

**Review by kpethtel — APPROVED** [review]
One question, but otherwise looks good.

File: modules/mobile/app/models/mobile/v0/contracts/claims_and_appeals.rb (hunk 2 above)
Comment (aherzberg) [reply modules/mobile/app/models/mobile/v0/contracts/claims_and_appeals.rb]: oops. yeah that slipped in when I was trying to debug something and forgot to remove it. fixed

**Review by kpethtel — APPROVED** [review]
👍

**Pull request merged**